models/dodo/arbitrum/dodo_arbitrum_schema.yml

```diff
@@ -160,16 +154,7 @@ models:
     description: >
         DODO Aggregation on Arbitrum.
     tests:
-      - dbt_utils.unique_combination_of_columns:
```

Comment:
why remove this test? we should keep and align with `unique_keys` values in model